src/gui/editorwidgets/qgsvaluerelationconfigdlg.cpp

```diff
@@ -78,6 +85,7 @@ QVariantMap QgsValueRelationConfigDlg::config()
   cfg.insert( QStringLiteral( "OrderByValue" ), mOrderByValue->isChecked() );
   cfg.insert( QStringLiteral( "FilterExpression" ), mFilterExpression->toPlainText() );
   cfg.insert( QStringLiteral( "UseCompleter" ), mUseCompleter->isChecked() );
+  cfg.insert( QStringLiteral( "CompleterMatchFromStart" ), mCompleterMatchFromStart->isChecked() );
```

Comment:
How would you feel about storing the integer Qt::MatchFlags value here instead of the bool? It would make it easier in future to expose other match options

Reply:
Good idea! I will do it.

Reply:
@nyalldawson done